### ISSUE TITLE
[Feature Request] Improved item names

### ISSUE THREAD

**Issue by Krutoy242** [raylras/zenscript-intelli-sense#131]
Please add this features to improve readability of item names

1. Remove `: ` in first of item name or make it configurable.
  > Any addition symbol already crowding and flooding code view. Also `:` symbol even add 2 additional dots, that actually unnecessary since item names already marked with other color and frame.

2. Strip Minecraft formatting codes from names
  > There is no use of symbols `§` in informative purpose. They need to be removed.

3. Make names italic or with another font.
  > I believe, that gray font still not enough, and brain still taking item names as part of the code. I believe, if this text will be italic, or even better - other font, it will be clearer to divide it from real code.

![image](https://github.com/raylras/zenscript-intelli-sense/assets/936618/74f721c8-d207-46a8-b85c-d7ea911ed05b)

**Comment by raylras** [comment]
1. > Remove `: ` in first of item name or make it configurable.

In short: I won't be making changes to it for now.

Based on what I've seen, most inlay hints start or end with specific symbols like `:`, `^`, `=`.
- Kotlin in IntelliJ IDEA
![image](https://github.com/raylras/zenscript-intelli-sense/assets/61815515/aaa64bbd-fc67-4e48-b70b-e99ccc5c140f)
- JavaScript in VS Code
![image](https://github.com/raylras/zenscript-intelli-sense/assets/61815515/4ed0e2e9-461c-4a1e-8fca-f504cf73722a)

To ensure **consistency**, I think it's best for hint names to follow that pattern as well, and `:` is the most suitable symbol.
![image](https://github.com/raylras/zenscript-intelli-sense/assets/61815515/4fbfc3b9-bb2b-43cb-9ad1-b655f33a2b0d)

How about make it configurable? 
The answer is: Yes we can, but it's not necessary. I want the plugin to be as simple as possible.

2. > Strip Minecraft formatting codes from names.

 I agree with you. It should be fixed by friendlyhj at https://github.com/friendlyhj/ProbeZS/commit/640a0aea7708069dd2e27c8a04a2b2a748c322da .

3. > Make names italic or with another font.

In short: No can do.

I understand your point. Inlay hints can be easily confused as part of the code，and this situation is not limited to ZenScript but applies to any language.

However, LSP does not support specifying styles like italics or different fonts for inlay hints.

**Comment by Krutoy242** [comment]
Ok, I got bout 2. and 3.

But about 1, I have a few arguments why `: ` shouldn't be:

1. The `:` should serve primarily as a "separator", a symbol between the _item_ and its _name_. But because all items are **already separated** by the `>` sign, it is an unnecessary symbol.
2. Sign `:` is not the most appropriate because it repeats the sign `:` inside `<mod:name>`. Also aggravates readability is that the : sign after <capture> is often used in objects `{ <item>: value }`.
3. Comparison with other languages is not really relevant here - I don't know about Kotlin, but in TypeScript variables are declared with `let name: type = value`, so JavaScript essentially copies this syntax - in this case, it's as if the gray `: type` is actually the executable part of the code.

In any case, it would be nice to add a format configuration.
